spectrocloud/provider.go

```diff
@@ -18,42 +18,50 @@ func New(_ string) func() *schema.Provider {
 				"host": {
 					Type:        schema.TypeString,
 					Optional:    true,
+					Description: "The Spectro Cloud API host url. Can also be set with the `SPECTROCLOUD_HOST` environment variable. Defaults to https://api.spectrocloud.com",
 					DefaultFunc: schema.EnvDefaultFunc("SPECTROCLOUD_HOST", "api.spectrocloud.com"),
 				},
 				"username": {
 					Type:        schema.TypeString,
 					Optional:    true,
+					Description: "The Spectro Cloud username. Can also be set with the `SPECTROCLOUD_USERNAME` environment variable.",
 					DefaultFunc: schema.EnvDefaultFunc("SPECTROCLOUD_USERNAME", nil),
 				},
 				"password": {
 					Type:        schema.TypeString,
 					Optional:    true,
 					Sensitive:   true,
+					Description: "The Spectro Cloud user password. Can also be set with the `SPECTROCLOUD_PASSWORD` environment variable.",
 					DefaultFunc: schema.EnvDefaultFunc("SPECTROCLOUD_PASSWORD", nil),
 				},
 				"api_key": {
 					Type:        schema.TypeString,
 					Optional:    true,
 					Sensitive:   true,
+					Description: "The Spectro Cloud API key. Can also be set with the `SPECTROCLOUD_API_KEY` environment variable.",
 					DefaultFunc: schema.EnvDefaultFunc("SPECTROCLOUD_APIKEY", nil),
 				},
 				"trace": {
 					Type:        schema.TypeBool,
 					Optional:    true,
+					Description: "Enable debugging. Can also be set with the `SPECTROCLOUD_TRACE` environment variable.",
 					DefaultFunc: schema.EnvDefaultFunc("SPECTROCLOUD_TRACE", nil),
 				},
 				"retry_attempts": {
 					Type:        schema.TypeInt,
 					Optional:    true,
+					Description: "Number of retry attempts. Can also be set with the `SPECTROCLOUD_RETRY_ATTEMPTS` environment variable. Defaults to 10.",
 					DefaultFunc: schema.EnvDefaultFunc("SPECTROCLOUD_RETRY_ATTEMPTS", 10),
 				},
 				"project_name": {
-					Type:     schema.TypeString,
-					Optional: true,
+					Type:        schema.TypeString,
+					Optional:    true,
+					Description: "The Spectro Cloud project name.",
 				},
 				"ignore_insecure_tls_error": {
-					Type:     schema.TypeBool,
-					Optional: true,
+					Type:        schema.TypeBool,
+					Optional:    true,
+					Description: "Ignore insecure TLS error. Defaults to false.",
```

Comment:
Ignore insecure TLS error for Spectro Cloud endpoint

Reply:
Thanks for clarifying 👍🏻